src/app/src/components/RegisterForm.jsx

```diff
@@ -0,0 +1,259 @@
+import React, { Component } from 'react';
+import { arrayOf, bool, func, shape, string } from 'prop-types';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import Grid from '@material-ui/core/Grid';
+
+import AppGrid from '../containers/AppGrid';
+import Checkbox from './inputs/Checkbox';
```

Comment:
It occurred to me that this should also have a ControlledCheckbox component so we can gradually remove the uncontrolled Checkbox. It's a pretty easy fix so I'll push a commit with that change.

Reply:
Handled in 
59836aa 

I also just deleted the existing uncontrolled components in 
f58af56